metricflow/cli/main.py

```diff
@@ -213,6 +218,13 @@ def tutorial(ctx: click.core.Context, cfg: CLIContext, msg: bool, skip_dw: bool,
     default=False,
     help="In the query output, show the query that was executed against the data warehouse",
 )
+@click.option(
+    "--display-plans",
```

Comment:
Can we please add this flag to the README and include an example output?

Reply:
There wasn't a section for CLI options in the README so I put it in the tutorial. Adding the image seems like it would take a lot of real estate.

Reply:
It would probably be better to add a example input/output in the docs somewhere rather than the README

Reply:
To me an image demonstrates how truly powerful this feature is. It makes me want to try it out. That said, it's your call!

Reply:
That's fair, especially since users need to install another package for this to work. Let me try it out.